packages/web-api-scan-runner/src/runner/runner.ts

```diff
@@ -92,7 +102,9 @@ export class Runner {
             this.logger.trackEvent('ScanRequestCompleted', undefined, { completedScanRequests: 1 });
         }
 
-        const fullPageScanResult = await this.onDemandPageScanRunResultProvider.updateScanRun(pageScanResult);
+        this.updateCombinedResults(fullPageScanResult.websiteScanIds);
```

Comment:
on each scan we generate combined axe result, we do not need to regenerate combined axe result for entire list. Here we may need only current scan id